### ISSUE TITLE
[BUG] API Document link in readme is broken.

### ISSUE THREAD

**Issue by trivialfis** [rapidsai/cudf#2745]
On branch 0.10 in the readme:

```
For additional examples, browse our complete API documentation, or check out our more detailed notebooks.
```

Link in `API documentation` is broken.

**Comment by cwharris** [comment]
Looks like we have documentation in multiple places, such as https://docs.rapids.ai and https://rapidsai.github.io/projects/cudf/en/0.9.0/index.html .

It looks like https://docs.rapids.ai is the latest, but I imagine we need to migrate / delete the old documentation, and clearly need to update references in existing code (lots of bad link in `notebooks` and `notebooks-extended` repos.

@kkraus14 Where is our prefered / latest / most up-to-date documentation hosted + source-controlled?

**Comment by kkraus14** [comment]
Fixed by #2769
